bin/install.sh

```diff
@@ -184,9 +184,6 @@ function install_ansible() {
 function run_ansible_playbook() {
     display_section "Run the Anthias Ansible Playbook"
 
-    sudo -u ${USER} ${SUDO_ARGS[@]} ansible localhost \
-        -m git \
-        -a "repo=$REPOSITORY dest=${ANTHIAS_REPO_DIR} version=${BRANCH} force=yes"
```

Comment:
Why was this removed?
(What are the issues that you've encountered before removing this?)

Reply:
This was a mistake on my part. I've removed this change from my code and it should no longer be a problem.